src/transformStateWithClones.js

```diff
@@ -7,7 +7,31 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const copyState = { ...state };
+  const array = [];
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        for (const key in action.extraData) {
+          copyState[key] = action.extraData[key];
+        }
+        break;
+      case 'removeProperties':
```

Comment:
add spaces between cases
```suggestion
        break;

      case 'removeProperties':
```